amd_openvx_extensions/amd_rpp/include/internal_publishKernels.h

```diff
@@ -122,6 +122,15 @@ vx_status Brightness_Register(vx_context);
 vx_status Copy_Register(vx_context);
 vx_status CropMirrorNormalize_Register(vx_context);
 vx_status Nop_Register(vx_context);
+vx_status Resize_Register(vx_context);
+vx_status Crop_Register(vx_context);
+vx_status ColorTwist_Register(vx_context);
+vx_status ResizeMirrorNormalize_Register(vx_context);
+vx_status SequenceRearrange_Register(vx_context);
```

Comment:
Lines 125 - 129 should not be a part of this PR why is it included here?

Reply:
removed.